tests/testapp/models.py

```diff
@@ -51,3 +51,12 @@ def __str__(self):
 
     class Meta:
         abstract = False
+
+
+class Library(models.Model):
```

Comment:
Hi,

It's not clear to me, why did we need a new model to test this, was there a challenge with updating the old models?

Reply:
@atb00ker we needed a model which does not contain the organization field, but it's parent model contains it. I thought it will be better to introduce new model, as all the present models contained organization field, and editing the already present model would fail some of the test cases.